### PR TITLE
Improve information on sapmachine.io

### PR DESCRIPTION
**Pull request by skateball**
Visual result can be visited at https://skateball.github.io/SapMachine/

### CODE REVIEW

**Comment by SapMachine** [comment]
Hello @skateball, I'm sorry, this pull request doesn't meet the expectations. The pull request description has an invalid format. Please have a look at https://github.com/SAP/SapMachine/wiki/Formal-Requirements-of-Pull-Requests .

**Comment by SapMachine** [comment]
Hello @skateball, I'm sorry, this pull request doesn't meet the expectations. The pull request description has an invalid format. Please have a look at https://github.com/SAP/SapMachine/wiki/Formal-Requirements-of-Pull-Requests .

File: index.md [hunk 1]
```diff
@@ -3,31 +3,25 @@ layout: default
...
-    
+SapMachine is available on a variety of Operating System/CPU architecture combinations, and it is certified to adhere to the Java SE standard.
+It can serve as a drop-in replacement for any JDK in Java-based applications, ranging from small desktop applications to high-performance, large-scale server applications.
 
 ## Download
 
+<!-- ToDo: Disable Short Term Support Releases -->
+<!-- ToDo: checkboxes hoch, entweder nur vor den Buttons oder ganz hoch -->
+<!-- ToDo: selects in eine Zeile -->
+
```
Comment (RealCLanger) [inline index.md:24]: Please remove  the ToDos

**Review by RealCLanger — CHANGES_REQUESTED** [review]
There are a few files contained in this PR which we should not add.

**Pull request closed without merging**